src/verifiers/documentStatus/types.ts

```diff
@@ -0,0 +1,18 @@
+import { Reason } from "../../types/error";
```

Comment:
are there any changes to review in this file and the `utils` ?

Reply:
no changes, just moving things around for reuse